orttraining/orttraining/python/training/_ortmodule_training_manager.py

```diff
@@ -5,6 +5,8 @@
 
 from . import _ortmodule_utils as _utils, _ortmodule_io as _io
 from ._ortmodule_graph_execution_manager import GraphExecutionManager, _run_forward
+from onnxruntime.capi import _pybind_state as C
+from . import _utils as U
```

Comment:
`U` is not descriptive. Please keep `_utils`

Reply:
Sure, how is C (above) anymore descriptive?

Reply:
C as in C API, a convention used by PyTorch, ORTrainer and also ORTModule.